cloudigrade/internal/views.py

```diff
@@ -508,3 +511,52 @@ class InternalAzureMachineImageViewSet(
         "created_at": ["lt", "exact", "gt"],
         "updated_at": ["lt", "exact", "gt"],
     }
+
+
+class InternalDailyConcurrentUsageViewSet(
```

Comment:
It looks like this class and `DailyConcurrentUsageViewSet` are copy-pasted from each other with only minor changes. We should consolidate them to stay DRY. Consider having one subclass the other and add helper methods for the relevant differences.